addons/sms/i18n/sms.pot

```diff
@@ -1245,11 +1199,23 @@ msgstr ""
 msgid "You do not have access to the message and/or related document."
 msgstr ""
 
+#. module: sms
+#. openerp-web
+#: code:addons/sms/static/src/js/fields_sms_widget.js:0
+#, python-format
+msgid "Your SMS Text Message must include at least one non-whitespace character"
```

Comment:
Keep only this change and the other one e.g. +1 415 555 0100 . Other changes (fields link, date) should not be committed (and actually depends on installed addons).